Make markup reviews

olegusanet87 opened the pull request:
Зроблена розмітка ревюс та загальна по коверс (список і тд. без підключених картинок). По ревюс зроблено джс (локально працює).

olegusanet87 committed:
Make markup reviews

Зроблена розмітка ревюс та загальна по коверс (список і тд. без підключених картинок). По ревюс зроблено джс (локально працює).

Jey2507 reviewed (changes requested):
Будь ласка оновіть гілку main, щоб всі нові дані були у вас в гілці, на всякий випадок збережіть свій код коли будете це робити!
Якщо не пам'ятаєте як оновити, зайдіть в трелло, там є матеріали, а в них відео-інструкція як користуватись GitHub Desktop

olegusanet87 committed:
Merge branch 'main' into feature-covers

olegusanet87 committed:
Make makkup reviews and functional

Зроблено розмітку  і функціонал, залишилось стилі кнопок поправити.  Секція коверс розмітка без підключених фото і стилів ( стилі прописані але фони не адекватні)

olegusanet87 committed:
Merge branch 'main' into feature-covers

olegusanet87 committed:
Add style button

olegusanet87 committed:
Change makup reviews-item-box

Jey2507 reviewed (changes requested):
1)стріть першу секцію, та як її буду робити я
2) заберіть фіксовану висоту карточок
3)заберіть непотрібні коментарі та почистіть код
4) замініть всі кольори на змінні, їх знайдете в root 
5) імена не мають бути капсом
6) в версії для ноута у вас не всі карточки влазять
7)стилізація кнопок не така як на макеті, підіть до Олени, у неї такі самі кнопки уже стилізовані і спитайте які в не стилі

olegusanet87 committed:
Check after teamlid review

Виправлено по зауваженням. Ти прийми Оленині пули і я в неї перегляну стилі і поправлю стилі на кнопках. В принципі я назвав класи так як в неї тільки можливо зараз  начика класу не така...

olegusanet87 committed:
Merge branch 'main' into feature-covers

olegusanet87 committed:
change button style

olegusanet87 committed:
Merge branch 'main' into feature-covers

Jey2507 committed:
upgrate section teamLead

Jey2507 reviewed (approved):
я зробив зміни, тепер все в порядку
